src/test/java/org/broadinstitute/hellbender/tools/exome/ExomeToolsTestUtils.java

```diff
@@ -34,42 +34,57 @@ protected static File getTestDataDir(){
      */
     protected final static SAMSequenceDictionary REFERENCE_DICTIONARY = SAMSequenceDictionaryExtractor.extractDictionary(REFERENCE_FILE);
 
+    protected static Comparator<SimpleInterval> createIntervalComparator() {
```

Comment:
I was wondering why SimpleInterval is not  a Comparable<SimpleInterval>  and I would need to define this comparator at all. @droazen any comments?